internal/http/auth.go

```diff
@@ -62,7 +62,7 @@ func (h authHandler) login(w http.ResponseWriter, r *http.Request) {
 
 	h.cookieStore.Options.HttpOnly = true
 	h.cookieStore.Options.SameSite = http.SameSiteLaxMode
-	h.cookieStore.Options.Path = h.config.BaseURL
```

Comment:
I don't think we should be setting `cookieStore` to /. A significant amount of our users host on baseurls and this will allow other applications on the machine to read the cookie.